ui/src/views/settings/modals/change-password-modal/index.tsx

```diff
@@ -1,15 +1,45 @@
+import { useMutation } from '@apollo/client'
 import { Button, HelpText, Input, Label, Modal, Toolbar } from '@mergestat/blocks'
 import { XIcon } from '@mergestat/icons'
-import React, { useCallback } from 'react'
-import { useUserSettingsSetState } from 'src/state/contexts/user-settings.context'
+import React, { ChangeEvent, useCallback, useState } from 'react'
+import { UpdateUserPasswordMutation } from 'src/api-logic/graphql/generated/schema'
+import { UPDATE_USER_PASSWORD } from 'src/api-logic/graphql/mutations/manage-users'
+import { useUserSettingsContext, useUserSettingsSetState } from 'src/state/contexts/user-settings.context'
+import { showSuccessAlert } from 'src/utils/alerts'
+import { TEST_IDS } from 'src/utils/constants'
 
 export const ChangePasswordModal: React.FC = () => {
+  const [{ usernameEdit }] = useUserSettingsContext()
   const { setShowChangePasswordModal } = useUserSettingsSetState()
 
+  const [password, setPassword] = useState<string>()
+  const [passwordConfirm, setPasswordConfirm] = useState<string>()
+  const [error, setError] = useState<string>('')
+
+  const [updatePassword] = useMutation(UPDATE_USER_PASSWORD, {
+    errorPolicy: 'all',
+    onCompleted: (data: UpdateUserPasswordMutation) => {
+      data.userMgmtUpdateUserPassword && showSuccessAlert('Password changed')
+    }
+  })
+
   const close = useCallback(() => {
     setShowChangePasswordModal(false)
   }, [setShowChangePasswordModal])
 
+  const handleChangePassword = () => {
+    setError('')
+
+    if ((password && password.length < 10) || (passwordConfirm && passwordConfirm.length < 10)) {
```

Comment:
let's remove this requirement for now - since we don't enforce it on the backend and we don't have it in the update password area of `User Management` either